invenio_opendata/testsuite/data/cms/cms-primary-datasets-Run2011A.xml

```diff
@@ -2529,11 +2529,13 @@
 
     <datafield tag="942" ind1=" " ind2=" ">
       <subfield code="e">Collision energy: 7TeV</subfield>
-      <subfield code="r">Run2011A</subfield>
     </datafield>
```

Comment:
If we are deleting `942 $r Run2011A` on purpose (?) then please note that the file `cms-eventdisplay-files-Run2011A.xml` also contains them...